pages/brand.js

```diff
@@ -270,9 +274,13 @@ const Page = ({ css }) => (
         href="https://github.com/hackclub/theme-starter"
         variant="outline"
         mb={3}
+        mr={3}
       >
         Theme Starter on GitHub →
       </Button>
+      <Button as="a" href="https://github.com/hackclub/css" sx={{ mb: 3 }}>
+        Theme CSS on GitHub →
```

Comment:
```suggestion
        CSS Theme on GitHub →
```

On https://toolbox.hackclub.com, it's titled "CSS Theme"; I'm not sure what the right name for it is.

Reply:
That's much more readable 👍